src/database/adapters/__init__.py

```diff
@@ -52,6 +54,24 @@ async def set_notion_config(token: str, user: KhojUser):
     return notion_config
 
 
+async def create_khoj_token(user: KhojUser, name="Secret Key"):
+    "Create Khoj API key for user"
+    token = f"kk-{secrets.token_urlsafe(32)}"
+    api_config = await KhojApiUser.objects.acreate(token=token, user=user, name=name)
```

Comment:
I agree this would help with UX intent of making tokens easier to remember. I can create a simple name generator function that maybe a step in that direction. But creating a proper unique, random, memorable name maybe more work right now then required?